frontend/app/src/utils/permissions.ts

```diff
@@ -0,0 +1,46 @@
+export type tPermission = {
+  view: string;
+  create: string;
+  update: string;
+  delete: string;
```

Comment:
string is too wide. Instead for better type inference we could use `"X" | "Y" | etc...`

Reply:
it will change soon to numbers, and I don't have the real values yet